src/main/java/com/levik/stealer/collect/repository/entity/PictureEntity.java

```diff
@@ -0,0 +1,37 @@
+package com.levik.stealer.collect.repository.entity;
+
+import com.levik.stealer.nasa.model.NasaImage;
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.LocalDateTime;
+
+@Setter
+@Getter
+@Table(name = "pictures")
+@Entity
+public class PictureEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", nullable = false)
+    private Long id;
+
+    @Column(name = "img_src", nullable = false)
+    private String imgSrc;
+
```

Comment:
Use @CreationTimestamp. It  is a convenient annotation that sets the field value to the current timestamp when the entity is first saved. And in that case, it will really be insertion time. When you delegate the responsibility of setting the creation timestamp to Hibernate, you ensure consistency across your application. Every entity with @CreationTimestamp will follow the same logic, ensuring uniformity in how creation dates are recorded.